src/app/(views)/managed-courses/[courseId]/modules/create/page.tsx

```diff
@@ -55,38 +60,110 @@ export default function CreateModule() {
   }
 
   return (
-    <CForm onSubmit={handleSubmit(onSubmit)}>
-      <CInputGroup>
-        <CFormLabel htmlFor="title">Title</CFormLabel>
-        <CFormInput id="title" {...register('title', { required: true })} />
-        {errors.title && <CFormText className="text-danger">This field is required</CFormText>}
-      </CInputGroup>
-      <CInputGroup>
-        <CFormLabel htmlFor="description">Description</CFormLabel>
-        <CFormInput id="description" {...register('description', { required: true })} />
-        {errors.description && (
-          <CFormText className="text-danger">This field is required</CFormText>
-        )}
-      </CInputGroup>
+    <>
```

Comment:
Do we need this commented code? if yes, put a comment above explaining why this is still here.